Content.Shared/_Sunrise/Weapons/Ranged/ConsumableAmmoComponent.cs

```diff
@@ -0,0 +1,40 @@
+using Robust.Shared.GameStates;
+
+namespace Content.Shared._Sunrise.Weapons.Ranged
+{
+    [RegisterComponent, AutoGenerateComponentState, NetworkedComponent]
+    public sealed partial class ConsumableAmmoComponent : Component
+    {
+        [AutoNetworkedField]
+        [DataField("currentCharges")]
+        public int CurrentCharges = 0; 
```

Comment:
```suggestion
        [DataField, ViewVariables, AutoNetworkedField]
        public int CurrentCharges; 
```

Датафилдам не нужно указывать строковое значение, если оно совпадает с названием в коде.

Еще сделай все аттрибуты в одном списке через запятую и добавь везде VV

А еще int по умолчанию 0, поэтому указывать это не нужно

Reply:
Про значение инт по умолчанию в курсе, тупо из-за того, что везде значения по умолчанию задал почти дописал